src/netspresso_trainer/models/backbones/experimental/darknet.py

```diff
@@ -2,19 +2,36 @@
 Based on the Darknet implementation of Megvii.
 https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/models/darknet.py
 """
-from typing import Dict, Optional, List
+
+from typing import Dict, Optional, List, Type
 
 from omegaconf import DictConfig
 import torch
 from torch import nn
 
-from ...op.custom import ConvLayer, CSPLayer, Focus, SPPBottleneck
+from ...op.custom import (
+    ConvLayer,
+    CSPLayer,
+    Focus,
+    SPPBottleneck,
+    Bottleneck,
+    BasicBlock,
+    DarknetBlock,
+)
 from ...utils import BackboneOutput
 from ..registry import USE_INTERMEDIATE_FEATURES_TASK_LIST
 
-__all__ = ['cspdarknet']
-SUPPORTING_TASK = ['classification', 'segmentation', 'detection', 'pose_estimation']
+__all__ = ["cspdarknet"]
+SUPPORTING_TASK = ["classification", "segmentation", "detection", "pose_estimation"]
+
+
+BLOCK_FROM_LITERAL: Dict[str, Type[nn.Module]] = {
+    "basicblock": BasicBlock,
+    "bottleneck": Bottleneck,
+    "darknetblock": DarknetBlock,
+}
```

Comment:
Do we need BasicBlock and Bottleneck for darknet?
I think ConvLayer and DarknetBlock are enough.